src/angucomplete-alt.js

```diff
@@ -521,10 +521,16 @@ angucompleteAltModule.directive(
                     for (i = 0; i < scope.localData.length; i++) {
                         match = false
 
-                        for (s = 0; s < searchFields.length; s++) {
-                            value = extractValue(scope.localData[i], searchFields[s]) || ''
-                            match = match || (value.toString().toLowerCase().indexOf(str.toString().toLowerCase()) >= 0)
-                        }
+                        if (searchFields.some(field => {
+                            let value = extractValue(scope.localData[i], field)
+                            if (
+                                value.toString().toLowerCase().includes(str.toString().toLowerCase()) ||
+                                str.toString().toLowerCase().includes(value.toString().toLowerCase())
+                            ) {
+                                return true
+                            }
+                        }))
+                            match = true
```

Comment:
it seems like there is no point in having this `match` variable set to `false` and then to `true` with no use in between :P 
btw I got an email about this PR, no idea why

Reply:
Good point. Removed it :)

Reply:
You're getting emails about this because you "watch it". If you want you can un-watch it :D (approve my PR first tho 😂)